rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -562,7 +562,7 @@
       "title": "Authors",
       "description": "Author(s) of the resource. Can be either persons or organisations.",
       "type": "array",
-      "minItems": 0,
+      "minItems": 1,
```

Comment:
See: https://github.com/rero/rero-ils/pull/611 ??

Reply:
This should not be addressed by this PR. It has been commited buy error.